ui/src/__tests__/components/role/__snapshots__/RoleMember.test.js.snap

```diff
@@ -83,7 +83,6 @@ exports[`RoleMember should render 1`] = `
 }
 
 .emotion-0 {
-  color: #3570f4;
```

Comment:
Note that rgb(53, 112, 244) that was added is equal to #3570f4 that was removed. The snapshot style didn't actually change...